core/src/main/scala/cromwell/core/actor/BatchActor.scala

```diff
@@ -45,19 +46,21 @@ object BatchActor {
 abstract class BatchActor[C](val flushRate: FiniteDuration,
                              val batchSize: Int) extends FSM[BatchActorState, BatchData[C]] with Timers {
   private var shuttingDown: Boolean = false
-  
+
   implicit val ec = context.dispatcher
   private val name = self.path.name
 
   override def preStart(): Unit = {
-    log.info("{} configured to flush with batch size {} and processHead rate {}.", name, batchSize, flushRate)
-    timers.startPeriodicTimer(ScheduledFlushKey, ScheduledProcessAction, flushRate)
+    log.info("{} configured to flush with batch size {} and process rate {}.", name, batchSize, flushRate)
+    if (flushRate != Duration.Zero) {
```

Comment:
I'm afraid to ask

Reply:
I wasn't sure what would happen if you start a timer every 0 second so in case I `if`ed it.. :/

Reply:
It certainly wouldn't be good; I was just worried you had actually seen this happening. 😄